_posts/2020-10-30-snowflake-dataOps.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: post
+title: DataOps with AWS and Snowflake
+date: 2020-10-27
+tags: devops, dataops, aws, snowflake, ci, cd, flyway
+author: Zainab Maleki
+image: img/blog/dataops/dataOps.jpg
+---
+
+<center><img src="/img/blog/dataops/dataOps.jpg" /></center><br/>
+
+# What is DataOps?
+
+DevOps practices have been discovered, utilised and perfected over the past few years to allow speed and agility in software engineering. Despite these advancements in the software engineering discipline, they have not been widely adopted in the data engineering domain. DataOps is a concept that applies DevOps practices to data life cycles to achieve speed and agility whilst delivering trusted and high-quality data. Like DevOps, DataOps minimises manual interventions by codifying data related changes and deployment through automated CI/CD tools.<br/><br/>
```

Comment:
Minor; I don't think you need the line breaks (i.e. `<br/>`). If there's a space between lines I think it treats them as separate paragraphs and puts a margin in between